render/render.js

```diff
@@ -91,10 +145,15 @@ module.exports = function($window) {
 		return element
 	}
 	function createComponent(vnode, hooks, ns) {
-		vnode.state = copy(vnode.tag)
-		
-		initLifecycle(vnode.tag, vnode, hooks)
-		vnode.instance = Node.normalize(vnode.tag.view.call(vnode.state, vnode))
+		if (typeof vnode.tag !== "function") {
```

Comment:
Should I add a `recycling` parameter to the `lifecycle.oninit` method and include this in it?